github/pulls_test.go

```diff
@@ -232,31 +233,63 @@ func TestPullRequestsService_Create_invalidOwner(t *testing.T) {
 }
 
 func TestPullRequestsService_Edit(t *testing.T) {
-	setup()
-	defer teardown()
+	tests := []struct {
+		input        *PullRequest
+		sendResponse string
+
+		wantUpdate   *pullRequestUpdate
```

Comment:
I was concerned that the order in which the struct fields would
be serialized is not deterministic but that doesn't appear to be
the case upon trial. Updated.